lsp-mode.el

```diff
@@ -505,7 +505,10 @@ the server has requested that."
                                     "[/\\\\]\\.deps\\'"
                                     "[/\\\\]build-aux\\'"
                                     "[/\\\\]autom4te.cache\\'"
-                                    "[/\\\\]\\.reference\\'")
+                                    "[/\\\\]\\.reference\\'"
+                                    ;; .Net Core build-output
+                                    "[/\\\\]bin/Debug$"
```

Comment:
please use `\\'` instead of `$` to prevent matching newline

Reply:
Done.